src/main/java/games/strategy/triplea/delegate/BattleDelegate.java

```diff
@@ -101,6 +101,12 @@ public void start() {
       m_needToAddBombardmentSources = false;
     }
     m_battleTracker.fightAirRaidsAndStrategicBombing(m_bridge);
+
+    final Collection<Territory> territories = m_battleTracker.getPendingBattleSites(false);
+    if (territories.size() == 1) {
+      final IBattle battle = m_battleTracker.getPendingBattle(territories.iterator().next(), false, BattleType.NORMAL);
+      battle.fight(m_bridge);
+    }
```

Comment:
Would be more likely to be accepted so in that sense it would be better. I'll do that tomorrow.